feelslikemonday/app/src/androidTest/java/com/example/feelslikemonday/DAO/FollowPermissionDAOTest.java

```diff
@@ -51,7 +51,71 @@ public void onCallback() {
                 signal.countDown();
             }
         });
+
         signal.await();
+    }
+
+    @Test
+    public void updateObject() throws InterruptedException {
+        final CountDownLatch signal = new CountDownLatch(1);
+        final FollowPermission followPermission = new FollowPermission("uTEST-bill");
 
+        DAO.createOrUpdate("uTEST-bill",followPermission,new VoidCallback(){
+            @Override
+            public void onCallback() {
```

Comment:
now that we are able to unfollow users. this class also needs a test to test the unfollowing of users

Reply:
I would consider that testing the un-follow activity rather than the DAO. The DAO only updates/creates without any additional logic.